src/Process/Process.php

```diff
@@ -40,15 +40,24 @@ public function getState(): string
         throw new \LogicException('This should never happen');
     }
 
-    public function __construct($commandline, ?string $cwd = null, ?array $env = null, $input = null, ?float $timeout = 60, array $options = [])
+    /**
+     * Process constructor.
+     * @param mixed $commandline
+     * @param string|null $cwd
+     * @param array|null $env
+     * @param null $input
+     * @param float|int|null $timeout
+     * @throws CommandExecutionException
+     */
+    public function __construct($commandline, ?string $cwd = null, ?array $env = null, $input = null, $timeout = 60)
```

Comment:
60.0? Or we change it in fromShellCommandline back to 60

Reply:
```suggestion
    public function __construct($commandline, ?string $cwd = null, ?array $env = null, $input = null, $timeout = 60.0)
```